src/pages/public_registration/ActionPanel.tsx

```diff
@@ -96,7 +98,7 @@ export const ActionPanel = ({
         sx={{ bgcolor: 'primary.main', px: '0.5rem' }}
         textColor="inherit"
         slotProps={{
-          indicator: { style: { backgroundColor: 'white', height: '0.4rem' } },
+          indicator: { style: { backgroundColor: theme.palette.secondary.main, height: '0.4rem' } },
```

Comment:
Må theme hook brukes her, kanskje?
Litt uheldig da det kan trigge unødvendige rerenders men
```suggestion
          indicator: { style: { bgcolor: 'secondary.main', height: '0.4rem' } },
```